core/templates/pages/topic-editor-page/services/topic-editor-routing.service.spec.ts

```diff
@@ -80,4 +82,13 @@ describe('Topic editor routing service', function() {
     $rootScope.$apply();
     expect(TopicEditorRoutingService.getActiveTabName()).toEqual('main');
   });
+
+  it('should navigate to skill editor', function() {
+    spyOn($window, 'open' ).and.callFake( function() {
```

Comment:
Delete space before ')', and after '('. Ditto below in multiple places.

Reply:
Ok fixed this.